ozaria/site/components/play/PageUnitMap/common/UnitMapLevelDot.vue

```diff
@@ -0,0 +1,178 @@
+        }
+      }
+    },
+    created () {
+      this.setLevelTypeAndIcon()
+      this.setLevelStatus()
+      this.setLevelConcepts()
+    },
+    mounted () {
+      $('.has-tooltip').tooltip({ html: true })
+    },
+    methods: {
+      setLevelTypeAndIcon () {
+        if (this.levelData.ozariaType === 'practice') {
+          this.levelType = 'Practice'
+          this.levelIcon['Complete'] = '/images/ozaria/unit-map/complete_practice.png'
+          this.levelIcon['Locked'] = '/images/ozaria/unit-map/locked_practice.png'
+          this.levelIcon['In Progress'] = '/images/ozaria/unit-map/unlocked_practice.png'
+        } else if (this.levelData.ozariaType === 'challenge') {
+          this.levelType = 'Challenge'
+          this.levelIcon['Complete'] = '/images/ozaria/unit-map/complete_challenge.png'
+          this.levelIcon['Locked'] = '/images/ozaria/unit-map/locked_challenge.png'
+          this.levelIcon['In Progress'] = '/images/ozaria/unit-map/unlocked_challenge.png'
+        } else if (this.isCutsceneLevel) {
+          this.levelType = 'Cutscene'
+          this.levelIcon['Complete'] = '/images/ozaria/unit-map/complete_cutscene.png'
+          this.levelIcon['Locked'] = '/images/ozaria/unit-map/locked_cutscene.png'
+          this.levelIcon['In Progress'] = '/images/ozaria/unit-map/unlocked_cutscene.png'
+        } else if (this.levelData.ozariaType === 'capstone') {
+          this.levelType = 'Capstone'
+          this.levelIcon['Complete'] = '/images/ozaria/unit-map/complete_capstone.png'
+          this.levelIcon['Locked'] = '/images/ozaria/unit-map/locked_capstone.png'
+          this.levelIcon['In Progress'] = '/images/ozaria/unit-map/unlocked_capstone.png'
+        } else if (this.levelData.type === 'intro') {
+          this.levelType = 'Intro'
+          this.levelIcon['Complete'] = '/images/ozaria/unit-map/complete_intro.png'
+          this.levelIcon['Locked'] = '/images/ozaria/unit-map/locked_intro.png'
+          this.levelIcon['In Progress'] = '/images/ozaria/unit-map/unlocked_intro.png'
+        } else {
+          // Using practice as the default values
+          this.levelType = 'Practice'
+          this.levelIcon['Complete'] = '/images/ozaria/unit-map/complete_practice.png'
+          this.levelIcon['Locked'] = '/images/ozaria/unit-map/locked_practice.png'
+          this.levelIcon['In Progress'] = '/images/ozaria/unit-map/unlocked_practice.png'
+        }
+      },
+      setLevelStatus () {
+        if (this.levelData.locked) {
+          this.levelStatus = 'Locked'
+        } else if (this.levelData.next) {
+          this.levelStatus = 'In Progress'
+        } else {
+          this.levelStatus = 'Complete'
+        }
+      },
+      setLevelConcepts () {
+        if ((this.levelData.concepts || []).length > 0) {
+          this.concepts = this.levelData.concepts
+        }
+      }
+    }
+  })
+</script>
+
+<template lang="pug">
+  div.level-dot(:style="levelDotPosition")
```

Comment:
Not sure, I think I started working on this before we formed the vue best practices, and hence just followed the legacy vue components in the codebase. Happy to change it to html, since its really small 😄